src/Controller/FeedController.php

```diff
@@ -28,15 +32,40 @@ public function index(DogCRUDService $dogService, PostRepository $postRepository
         
         $posts = $postRepository->findAll();
 
+        $comments = $commentRepository->findAll();
+
         return $this->render('feed/feed.html.twig', 
         [
             'posts' => $posts,
             'currentDogUser' => $postAuthor,
+            'comments' => $comments,
             ]
         );
     }
 
-    public function createPost(Request $request, PostCRUDService $postService, DogCRUDService $dogService): Response 
+    public function createComment(Request $request, EntityManagerInterface $em , Post $post) : Response
+    {
+        $comment = new Comment();
+        $comment->setPost($post);
+        $form = $this->createForm(CommentFormType::class, $comment);
+
+        $dogUser = $this->getUser();
+        $comment->setDog($dogUser);
+        $form->handleRequest($request);
+        
+        if($form->isSubmitted() && $form->isValid()) {
+            $em->persist($comment);
+            $em->flush();
+            return $this->redirectToRoute('feed');
+        }
+
+        // Rendering the view if the form has not been submitted
+        return $this->render('feed/comment/add-comment.html.twig', [
+            'form' => $form->createView(),
+        ]);
+    }
+
```

Comment:
This should be in a service too. Let's refactor this stuff out when we meet next